dev/com.ibm.ws.microprofile.opentracing/bnd.bnd

```diff
@@ -0,0 +1,48 @@
+#*******************************************************************************
+# Copyright (c) 2017 IBM Corporation and others.
+# All rights reserved. This program and the accompanying materials
+# are made available under the terms of the Eclipse Public License v1.0
+# which accompanies this distribution, and is available at
+# http://www.eclipse.org/legal/epl-v10.html
+#
+# Contributors:
+#     IBM Corporation - initial API and implementation
+#*******************************************************************************
+-include= ~../cnf/resources/bnd/bundle.props
+bVersion=1.0
+
+javac.source: 1.8
+javac.target: 1.8
+
+Bundle-Name: com.ibm.ws.microprofile.opentracing
+Bundle-SymbolicName: com.ibm.ws.microprofile.opentracing
+Bundle-Description: MicroProfile.OpenTracing 1.0, version ${bVersion}
+
+WS-TraceGroup: OPENTRACING
+
+-dsannotations: com.ibm.ws.microprofile.opentracing.OpenTracingFilterHelper
+
+Import-Package: *
+
+Export-Package: \
+    com.ibm.ws.microprofile.opentracing, \
+    com.ibm.ws.microprofile.opentracing.cdi
+
+Require-Capability: osgi.ee;filter:="(&(osgi.ee=JavaSE)(version=1.8))"
+
+-buildpath: \
+	io.opentracing:opentracing-api;version=0.30.0, \
```

Comment:
is this just a re-add from a previously removed file? is it correct to have this depend directly on io.opentracing and not have it wrapped in something?

Reply:
it's doing the same thing that the com.ibm.ws.opentracing bundle is doing, so if it's wrong it's now wrong in two places.  will ask folks that have working on the code previously why that was.